front-end-master/Projects/projects.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+	<!-- Bootstrap CSS -->
+	<link rel="stylesheet" href="../common_files/css/bootstrap.min.css">
+	<link rel="stylesheet" href="../common_files/css/navbar.css">
+	<link rel="stylesheet" href="../common_files/css/footer.css">
+	<link rel="stylesheet" href="./projects.css">
+	<link href="https://fonts.googleapis.com/css?family=Pacifico|Patua+One|Roboto|Roboto+Condensed|Lato|Raleway:700|Raleway:500|arvo" rel="stylesheet">
+
+
+
+	<!-- jquery -->
+	<script src="../common_files/js/jquery-3.1.1.js"></script>
+	<title>Our Projects</title>
+</head>
+
+
+<body>
+	<nav class="navbar nav-container">
```

Comment:
At some point in the future, we need to refactor the Header into its own component.

Reply:
yes. thats why i tried to keep navbar html and css as modular as possible